dialogue-apache-hc5-client/build.gradle

```diff
@@ -24,4 +24,6 @@ dependencies {
     testImplementation project(':dialogue-serde')
     testImplementation 'org.awaitility:awaitility'
     testImplementation 'org.junit.jupiter:junit-jupiter'
+    testRuntimeOnly 'org.apache.logging.log4j:log4j-slf4j-impl'
+    testRuntimeOnly 'org.apache.logging.log4j:log4j-core'
```

Comment:
With the logging implementation in place, the reported warnings were very obvious in tests.